src/main/java/models/Game.java

```diff
@@ -0,0 +1,72 @@
+package models;
+
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Game {
+
+    private static final Game GAME_INSTANCE = new Game();
+
+    private ArrayList<Player> allPlayers;
+    private ArrayList<Team> allTeams;
```

Comment:
Could also be Maps mapped by name. May or may not be useful for accessing players or teams by name.